### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by RomanPerepelkin**
- [DEMO LINK](https://RomanPerepelkin.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://RomanPerepelkin.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:8080/"
+    method="post"
+    class="bar-form"
+    name="bar-form">
```
Comment (vlad-demchuk) [inline src/index.html:17]:
```suggestion
    <form
      action="http://localhost:8080/"
      method="post"
      class="bar-form"
      name="bar-form"
    >
```
check your code style

Comment (vlad-demchuk) [reply src/index.html:17]: check all such cases

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.big-field {
+  position: relative;
+  height: 70px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.small-field {
```
Comment (vlad-demchuk) [inline src/style.css:30]: no need to repeat all styles and use `big-field` and `small-field`

checklist
image

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+input::placeholder {
```
Comment (vlad-demchuk) [inline src/style.css:92]: do not style tags

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
try to remove default outline
image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
```
Comment (ivadimko) [inline src/style.css:9]: Missing font-weight property for the Avenir Heavy font. Add `font-weight: 600;` or any appropriate value.

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  top: 25px;
+}
+
+.small-image {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
```
Comment (ivadimko) [inline src/style.css:51]: The 'bottom' property is not necessary for the .small-image class. Consider removing it.

File: src/index.html [hunk 6]
```diff
@@ -11,6 +11,29 @@
...
+          name="fist-field"
+          placeholder="Try &#34Los Angeles&#34">
+        <img src="./images/Search.svg"
+        alt="image"
+        class="big-image">
+      </div>
+      <div class="small-field" data-qa="small">
+        <input type="text"
+          class="small-search-bar"
+          name="fist-field"
```
Comment (ivadimko) [inline src/index.html:31]: Typo in the attribute name. Change `name="fist-field"` to `name="first-field"`

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,29 @@
...
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:8080/"
+    method="post"
+    class="bar-form"
+    name="bar-form">
+      <div class="big-field" data-qa="big">
+        <input type="text"
+          class="big-search-bar"
+          data-qa="keypress"
+          name="fist-field"
```
Comment (ivadimko) [inline src/index.html:22]: Typo in the attribute name. Change `name="fist-field"` to `name="first-field"`

File: src/index.html (hunk 6 above)
Comment (ivadimko) [inline src/index.html:31]: Both input fields have the same name attribute. Consider using unique names for each input field.

**Pull request closed without merging**